e2e/test/scenarios/organization/moderation-question.cy.spec.js

```diff
@@ -142,12 +142,12 @@ describeEE("scenarios > saved question moderation", () => {
 
       cy.findByPlaceholderText("Search…").type("orders{enter}");
       // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-      cy.findByText("Orders, Count").icon("verified");
```

Comment:
This change exposed a misleading pattern/usage from before.

Icon wasn't really chained in this line:
`cy.findByText("Orders, Count").icon("verified");`

This was the equivalent of:
```js
cy.findByText("Orders, Count");
cy.icon("verified");
```